pkg/backend/display/object_diff.go

```diff
@@ -1268,3 +1281,31 @@ func (p *propertyPrinter) translateYAMLValue(v interface{}) (interface{}, bool)
 		return v, true
 	}
 }
+
+// if string exceeds three lines or is >150 characters, truncate and add "..."
+func (p *propertyPrinter) truncatePropertyString(propertyString string) string {
+	const (
+		contextLines  = 3
+		maxLineLength = 150
+	)
+
+	lines := strings.Split(propertyString, "\n")
+	numLines := len(lines)
+	if numLines > contextLines {
+		numLines = contextLines
+	}
+
+	for i := 0; i < numLines; i++ {
+		if len(lines[i]) > maxLineLength {
+			lines[i] = lines[i][:maxLineLength] + "..."
+		}
+	}
+
+	if numLines == 1 {
+		return lines[0]
+	} else if len(lines) <= contextLines {
+		return strings.Join(lines, "\n")
+	}
```

Comment:
Joining a single line just returns it. 
```suggestion
	if len(lines) <= contextLines {
 		return strings.Join(lines, "\n")
 	}
```